languages/en.php

```diff
@@ -1272,19 +1272,30 @@
 	'generic_comment:on' => '%s on %s',
 	'generic_comments:latest:posted' => 'posted a',
 
-	'generic_comment:email:subject' => 'You have a new comment!',
-	'generic_comment:email:body' => "You have a new comment on your item \"%s\" from %s. It reads:
+	'generic_comment:notification:owner:subject' => 'You have a new comment!',
+	'generic_comment:notification:owner:summary' => 'You have a new comment!',
+	'generic_comment:notification:owner:body' => "You have a new comment on your item \"%s\" from %s. It reads:
 
+%s
 
+To reply or view the original item, click here:
 %s
 
+To view %s's profile, click here:
+%s
 
-To reply or view the original item, click here:
+Please do not reply to this email.",
```

Comment:
Let's remove these from notifications. We have mailgun inbound parsing and these don't make sense. Sites can add this to their templates.